rust/azure_iot_operations_services/src/azure_device_registry.rs

```diff
@@ -1108,63 +1104,29 @@ impl From<adr_name_gen::AssetDatasetDataPointSchemaElementSchema> for DatasetDat
     }
 }
 
-impl From<adr_name_gen::AssetDatasetDestinationSchemaElementSchema> for DatasetDestination {
-    fn from(value: adr_name_gen::AssetDatasetDestinationSchemaElementSchema) -> Self {
+impl From<base_client_gen::DatasetDestination> for DatasetDestination {
+    fn from(value: base_client_gen::DatasetDestination) -> Self {
         DatasetDestination {
             configuration: value.configuration.into(),
             target: value.target.into(),
         }
     }
 }
 
-impl From<adr_name_gen::DefaultDatasetsDestinationsSchemaElementSchema> for DatasetDestination {
```

Comment:
There's a few From impls that you've removed - is there a change that makes them not needed, or were they not needed before as well?

Reply:
There is no longer a bespoke generated type for a Default Destination. Destinations are now standardized